modules/social_features/social_comment/graphql/social_comment_schema_extension.base.graphqls

```diff
@@ -1,3 +1,23 @@
+"""
+Commentable interface that contains fields that present in the commentable entity.
+"""
+interface Commentable {
```

Comment:
`Commentable` interface added to `social_comment` module, so not sure if there's any reason to create another PR for this since we already have a lot of PR's :|